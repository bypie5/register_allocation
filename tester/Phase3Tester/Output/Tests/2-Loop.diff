--- SelfTestCases/2-Loop.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Output/Tests/2-Loop.out	2020-02-24 15:38:10.000000000 -0800
@@ -1,7 +1 @@
-1
-2
-4
-8
-16
-32
-64
+Program has no "Main" function.
