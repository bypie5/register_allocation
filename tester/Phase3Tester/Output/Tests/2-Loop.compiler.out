func Main [in 0, out 0, local 0]
$s0 = 1
goto :end
begin:
PrintIntS($s0)
$s0 = MulS($s0 2)
end:
$s1 = LtS($s0 100)
if $s1 goto :begin
ret

